executor/backup.go

```diff
@@ -63,18 +75,26 @@ func (b *BackupExecutor) Execute() error {
 		return nil
 	}
 
+	b.setConditionsUnknown(
+		ConditionJobCreated,
+		ConditionRoleBindingReady,
+		ConditionRoleReady,
+		ConditionServiceAccountReady,
+		ConditionCleanupSucceeded,
+	)
+
 	err := b.createServiceAccountAndBinding()
 	if err != nil {
 		return err
 	}
 
-	job, err := job.GetGenericJob(b.Obj, b.Config)
+	genericJob, err := job.GetGenericJob(b.Obj, b.Config)
```

Comment:
Try to prevent changes irrelevant to the task at hand. I is a good habit that have saved me a lot of potential merge conflicts.

Reply:
Thanks for the advice. I try to adhere to [the Boyscout Rule](https://deviq.com/boy-scout-rule/): _Leave your code better than you found it._ Here, `job` was colliding with an import named `job`. Since I've discovered it, I've changed it.